jsHelper/spicetifyWrapper.js

```diff
@@ -1543,15 +1562,26 @@ Spicetify.Playbar = (function() {
                 });
             })();
 
+            function changelogs() {
```

Comment:
Use a variable instead of a function, function is only used once and should not access properties outside of its scope.